# Engine id to create
string id

# Path to rules file in the file system that DecisionService runs
string file
---
# Success == 1, Failure == 0
int32 success
